packages/opc-feedback/declaration.d.ts

```diff
@@ -9,17 +9,25 @@ type ErrorList = {
 type ExperienceList = {
   name: string;
   assetUrl: string;
-  errorList: ErrorList[];
 }
 
 type TemplateView = {
-  title: string;
+  buttonName: string;
```

Comment:
@riginoommen Can you provide a better name for this? Might confuse the user what this button means

Reply:
updated this